pkgs/by-name/ic/icestudio/package.nix

```
@@ -0,0 +1,96 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildNpmPackage,
+  nwjs,
+  makeWrapper,
+  python3,
+  python3Packages,
+  fetchpatch,
+  fetchurl,
+}:
+
+let
+  # Use unstable because it has improvements for finding python
+  version = "unstable-2024-07-11";
```

Unstable package versions should start with `0` (see [Package Naming](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming))

```suggestion
  version = "0-unstable-2024-07-11";
```